tool/tsh/common/db.go

```diff
@@ -817,7 +817,7 @@ func onDatabaseConnect(cf *CLIConf) error {
 // is active in profile and no labels or predicate query are given.
 // Otherwise, the ListDatabases endpoint is called.
 func getDatabaseInfo(cf *CLIConf, tc *client.TeleportClient) (*databaseInfo, error) {
-	haveSelectors := len(tc.Labels) > 0 || tc.PredicateExpression != ""
+	haveSelectors := tc.DatabaseService != "" || len(tc.Labels) > 0 || tc.PredicateExpression != ""
```

Comment:
isn't this always true?

Reply:
you can `tsh db login <db>` followed by `tsh db connect` without args, so this just preserves that behavior to select the active cert